.github/workflows/publish-site.yaml

```diff
@@ -3,7 +3,7 @@ name: Publish Site
 on:
   workflow_dispatch:
   push:
-    branches: [main]
+    branches: [main,'website/**']
```

Comment:
Should we remove `main`? Since this workflow is in both branches `main` and `website`, I am a bit cautious.